.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Ich glaub wir müssten mal was zu gitignores sagen...

Hilfreiche Seite: [gitignore.io](https://gitignore.io/)

Reply:
Ja, hatte erst gedacht, node_modules in der .gitignore alleine reichen schon. 
Die Erläuterung gerade war echt nützlich. :D